ClientApp/viking-vault/src/UserDashboard/DownloadTypeForm.css

```diff
@@ -0,0 +1,40 @@
+.download-icon {
+    height: 7%;
+    width: 7%;
+    position: relative;
+    border: none;
+}
+
+.download-button:focus,.download-button:active {
+    outline: none !important;
```

Comment:
`!important` is like dropping an atomic bomb 💣☢to kill a rat 🐀, it's really really best to avoid it. See a bit about CSS selector specificity to see how you could avoid this

Reply:
I deleted those and it seems to work just the same... Thank you!